src/main/java/com/mraof/minestuck/event/ClientEventHandler.java

```diff
@@ -77,11 +90,30 @@ public static void addCustomTooltip(ItemTooltipEvent event)
 	@SubscribeEvent
 	public static void onFogRender(EntityViewRenderEvent.FogDensity event)
 	{
-		if (event.getInfo().getFluidState().getFluid() == MSFluids.ENDER.get())
+		if (event.getInfo().getFluidState().getFluid() == MSFluids.ENDER.get() || event.getInfo().getFluidState().getFluid() == MSFluids.FLOWING_ENDER.get())
```

Comment:
If you have the time, I'd suggest using the ender fluid tag since you're checking both fluid types